ResourceManager/GitExtensionsControlInitialiser.cs

```diff
@@ -8,38 +10,60 @@ namespace ResourceManager
 {
     internal sealed class GitExtensionsControlInitialiser
     {
+        private static bool? _isDesignMode;
         private readonly ITranslate _translate;
 
-        /// <summary>indicates whether the initialisation has been signalled as complete.</summary>
+        // Indicates whether the initialisation has been signalled as complete.
         private bool _initialiseCompleteCalled;
```

Comment:
❔ Shall private members be hidden from xmldoc?

Reply:
Xml-docs are generally applied to public API, as they are used to generate documentation. Annotations of non public API is optional. In this specific case I didn't think it was necessary